packages/strapi-plugin-graphql/services/type-builder.js

```diff
@@ -232,6 +239,7 @@ module.exports = {
       input ${inputName} {
 
         ${Object.keys(model.attributes)
+          .filter(attributeName => isNotPrivate(model, attributeName))
```

Comment:
avoid creating a new method and use `!isPrivateAttribute(model, attributeName)`

Reply:
same here